### ISSUE TITLE
Update UVIS User's Guide from PDS3 to PDS4

### ISSUE THREAD

**Issue by mace-space** [SETI/pds-data-projects#32]
In migrating to PDS4, there are some differences in terminology, file naming, and directory structure that need to be documented for the UVIS User’s Guide (which was originally written for PDS3). A brief document detailing these changes should be provided as an addendum to the UVIS User’s Guide and referenced in the relevant PDS4 labels.

Examples include (but are not limited to):	

- In PDS3, the data are divided into different volumes numbered from 1 to 60 (e.g. /path/to/COUVIS_0xxx/COUVIS_0001/DATA/D1999_007/FUV1999_007_16_57.DAT), whereas in PDS4 the data are placed into bundles (cruise and saturn) and then collections (raw data, calibration, document etc) and then the data products are specified by date (e.g. /path/to/cassini_uvis_euvfuv_cruise/data_raw/1999/1999_007/fuv1999_007_16_57.dat)
    - Note different filenames, e.g. EUV2005_172_03_35.DAT in PDS3, but euv2005_172_03_35.dat in PDS4
    - Statements like “Generally, each COUVIS directory consists of six separate directories: CALIB, CATALOG, DATA, DOCUMENT, INDEX, and SOFTWARE” need to be updated to reflect the PDS4 directory structure.
- PDS3 and PDS4 attributes use different cases (PDS3 attributes are all uppercase)
- PDS4 labels use file extension .xml, whereas PDS3 labels use .LBL
- The use of XML in PDS4 affects all the label examples. For example, in PDS3 the axes are specified by the attributes AXIS_NAME and CORE_ITEMS as tuples:
 e.g. 
> AXIS_NAME = (BAND, LINE, SAMPLE)
> CORE_ITEMS = (1024, 64, 163)

Whereas in PDS4 this is described by an Axis_Array element 
>              <Axis_Array>
>                  <axis_name>Sample</axis_name>
>                  <elements>$CORE_ITEMS[2]$</elements>
>                  <sequence_number>1</sequence_number>
>              </Axis_Array>
>              <Axis_Array>
>                  <axis_name>Line</axis_name>
>                  <elements>$CORE_ITEMS[1]$</elements>
>                  <sequence_number>2</sequence_number>
>              </Axis_Array>
>              <Axis_Array>
>                  <axis_name>Band</axis_name>
>                  <local_identifier>wavelength-axis</local_identif    ier>
>                  <elements>$CORE_ITEMS[0]$</elements>
>                  <sequence_number>3</sequence_number>
>              </Axis_Array>  

- In PDS4, the document collection contains uvis-instrument-description.txt, which in PDS3 was named UVISINST.CAT in the CATALOG folder.

**Comment by mace-space** [comment]
Following recent discussion with Matt, I will convert the UVIS User's Guide from PDS3 to PDS4. This will be a separate full Users Guide and not just an addendum.

@matthewtiscareno,  as well as creating a PDS4 Users Guide, I think I need to do the following?:

1. Add this PDS4 UVIS Users Guide to the Cassini Documents bundle that Lyle is curating
2. Check with Lyle whether he or I should reserve a DOI for it (by emailing Ron Joyner)
3. Add this PDS4 version to the EUV/FUV bundle (in the Saturn document collection, and add an Internal_Reference in the labels)

Note that the PDS3-version UVIS Users Guide was archived in PDS4  in the "misc" bundle: https://pds.nasa.gov/data/pds4/misc/document_cassini/Cassini_UVIS_Users_Guide_20180706.xml. The label does not include DOI details although it has one: [10.17189/1522470](https://doi.org/10.17189/1522470). This should be updated, I think Jordan may be correct person to contact about this.

**Comment by matthewtiscareno** [comment]
Right, I had forgotten that there is a Cassini Documents bundle.  But it's not published yet, is it?  We are ready to publish the EUV/FUV bundle, so perhaps we should simply add this to that bundle instead?

**Comment by mace-space** [comment]
The Cassini Documents bundle has not been published yet. It's certainly possible to add the PDS4 UVIS Users Guide to the EUV/FUV bundle, and there are already several Cassini documents that have already been archived outside of the Cassini Documents bundle (for example, the PDS3 Cassini UVIS Users Guide is in the `misc` bundle, the Cassini ISS Narrow-Angle Camera Description is in the `cassini_iss_saturn` bundle). However, we've already planned to put many of the UVIS documents in the Cassini Documents bundle:

- urn:nasa:pds:cassini_documents:uvis:uvis_instrument_description_catalog
- urn:nasa:pds:cassini_documents:multi-inst:cassini_final_report_vol1_full
- urn:nasa:pds:cassini_documents:multi-inst:cassini_final_report_vol1_sec3_icysatellites
- urn:nasa:pds:cassini_documents:multi-inst:cassini_final_report_vol1_sec3_uvis
- urn:nasa:pds:cassini_documents:uvis:uvis_pds3_sis
- ..etc

**Comment by matthewtiscareno** [comment]
I see.  Then your plan seems good, unless you have questions about any part of it.

**Comment by mace-space** [comment]
Sounds good.  How's this for a LID? 

> urn:nasa:pds:cassini_documents:uvis:uvis_pds4_user_guide

I can email Lyle about the update and check that he'll be reserving a DOI for it.

**Comment by matthewtiscareno** [comment]
Great

**Comment by mace-space** [comment]
Separately, I'll email Jordan about the PDS3 UVIS Users Guide label missing a DOI [10.17189/1522470](https://doi.org/10.17189/1522470) (I think he oversaw the misc bundle?). The PDF linked in "SEARCH/ACCESS DATA"  is the old version with the sidebar because of the 'track changes' setting.

**Comment by matthewtiscareno** [comment]
Thanks, @mace-space

**Comment by mace-space** [comment]
I made a duplicate of the PDS UVIS User Guide and then edited it with Track Changes so you can see the modifications I've made. Shared via Dropbox (UVIS-Users-Guide-PDS4-mjtm.docx)

**Comment by mace-space** [comment]
Related: I sent a reminder to Jordan and Richard about some minor issues with the PDS3 version of the UVIS Users Guide (missing DOI in the label, and pointed to the most recent version on Viewmaster (without Track Changes)).

**Comment by mace-space** [comment]
Following discussion with @mit3ch, we think we should proceed with updating the UVIS Users Guide for PDS4 so that it can be included in the EUV/FUV bundles. Mitch thinks the PDS3 version of the UVIS Users Guide should **not** be included in the migrated EUV/FUV bundles at all. 

We can leave the HDAC, HSP, and any other sections of the UVIS Users Guide that cannot be updated at this stage to subsequent (major) version updates, so that they do not hold up the EUV/FUV migration.

We need to reserve a DOI for this PDS4 version of the UVIS Users Guide. Mitch points out that the DOI can be associated with a LIDVID (not just a LID) and in those cases the DOI has to be updated alongside the updates to the document. Note that for a major version update, it is recommended to assign a new DOI – see https://support.datacite.org/docs/versioning. So when it comes to archiving HDAC, HSP, we will need to assign new DOIs for this PDS4 UVIS Users Guide.

@mit3ch will check his DOI WG notes to confirm that the unique identifier was required to be a LIDVID (not a LID).

Related: I don't think EN have fixed the minor issues with the PDS3 version of the UVIS Users Guide (`urn:nasa:pds:misc:document_cassini:pds3_cassini_uvis_ug_20180706`, which was archived in the misc bundle), so I plan to email them.

@matthewtiscareno, what do you think?

**Comment by matthewtiscareno** [comment]
Is this a change from the plan articulated at the top of this issue? 

Are you thinking of any changes other than those mentioned at the top of this issue?

Does the plan at the top of this issue contemplate leaving the PDS3 user guide in place but with an addendum describing the changes for PDS4 migration?  If you are considering more thoroughgoing changes, I'm willing to discuss it but I'm concerned about this turning into a big project that bogs down the migration.

What did we do for the ISS and VIMS user guides?  Is there a reason not to do the same thing for UVIS that we did for ISS and VIMS?

**Comment by matthewtiscareno** [comment]
You're correct that there is no VIMS Users Guide.  Apologies for bringing that up.

If we have an editable version of the UVIS Users Guide and if you're keen to do it and if it wouldn't be much effort, then please feel free to go ahead. 

On the other hand, if any of those three things is not the case, then I might say it's better (though still imperfect) to keep the PDS3 guide as it is.  I just don't want us to get bogged down on this.

If you do this, then please use Track Changes, and we might want to send that to our UVIS team member friends.

**Comment by mace-space** [comment]
The PDS3 UVIS Users Guide has a DOI. I think I should also reserve/register a DOI for the PDS4 version

**Comment by mace-space** [comment]
@matthewtiscareno, I have updated the draft with tracked changes in [the shared-mia-matt Dropbox folder](https://www.dropbox.com/scl/fi/2pfkjxrhdi8skc1ufz6ex/UVIS-Users-Guide-PDS4-mjtm_mst.docx?rlkey=s2m3qujrve8r836kzkthd795p&dl=0). Please would you review? I have some questions that I have asked in embedded comments in the doc.

**Comment by matthewtiscareno** [comment]
I sent @mace-space a version with some comments, only pertaining to the introduction.

I think this is definitely ready to move forward to peer review.  It strikes me that Michael Aye would be very well positioned to review this document and indeed the entire delivery.

**Comment by mace-space** [comment]
Matt followed up with Michael Aye in June 2026 about reviewing the document, but he said:

> Regarding the UVIS manual, I don’t think it’s wise that I spend any time on it at this moment as things are in flux on our side.
> I believe the PDS4 input is sufficiently well described for now, but we certainly will need to update it down the road.

I have essentially finished updating the PDS4 UVIS EUV/FUV User's Guide, see [this version](https://www.dropbox.com/scl/fi/mjdwrm0zz551jli2qhip9/UVIS-Users-Guide-PDS4-120626-mjtm.docx?rlkey=85pr3tqkalzmyvitmiafj3qid&dl=0).

Here's an overview of my changes:

- Added a paragraph to the start explaining about the unchanged IDL code. Also, added a note in a relevant later section so that reader is reminded.
- Checked that all the comments left in the document were resolved. If I thought they were, then I clicked "Resolve" (you can still view the comment), and if I didn't, then I left it unresolved for us to discuss further.
- Checked entire doc for (US) spelling and grammar.
- Fixed all the typos and formatting issues that I could find. 
- Updated Table of Contents page numbering to reflect updates.
- There was a comment on page 33 left by Mitch, which says that we may need to flesh out the discussion of "evil" pixels and NaNs. I wasn't sure what he had in mind, I think it's okay to leave the description as it is, with the missing_constant note?

Next step is to accept all changes to produce a clean version.

**Comment by mace-space** [comment]
Related to #114, we still need to produce PDS4 UVIS HSP User's Guide and PDS4 UVIS HDAC User's Guide. 

I think we will need to work with Josh Colwell et al to update Chapter 6 of the PDS3 UVIS User's Guide to PDS4 ("6. Cassini UVIS Ring and Icy Satellite Stellar Occultation Data"). This will also involve expanding the content of this chapter to include details for the full-res (unbinned) derived occ data that Lizzie Faulkner, Josh Colwell, and Richard Jerousek are working on. 

Note, we plan to include chapters 1, 2, and 3.1 of the PDS4 UVIS EUV/FUV User's Guide in each of the HDAC and HSP versions as they provide relevant information.

**Comment by mace-space** [comment]
Final version of PDS4 UVIS EUV/FUV User's Guide can be found [here](https://www.dropbox.com/scl/fi/c175tmi9cz16ugxmhnm42/UVIS-Users-Guide-PDS4-final_v1_clean_pdfa.pdf?rlkey=ve4mjz6kt0ebxblo06rrj1uvq&dl=0).
